pkgs/top-level/aliases.nix

```
@@ -1462,6 +1465,7 @@ mapAliases ({
   tftp_hpa = throw "'tftp_hpa' has been renamed to/replaced by 'tftp-hpa'"; # Converted to throw 2022-02-22
   thunderbird-68 = throw "Thunderbird 68 reached end of life with its final release 68.12.0 on 2020-08-25";
   thunderbird-bin-68 = thunderbird-68;
+  thunderbird-wayland = thunderbird;
```

```suggestion
  thunderbird-wayland = thunderbird; # Added 2022-11-15
```

pushing an update in a second.